packages/web-component-tester/runner/webserver.ts

```diff
@@ -80,37 +67,84 @@ export function webserver(wct: Context): void {
     // npm, the wct-browser-legacy package may be used, so we test for that
     // package and will use its "browser.js" if present.
     let browserScript = 'web-component-tester/browser.js';
-    if (options.npm) {
-      try {
-        const wctBrowserLegacyPath =
-            path.join(options.root, 'node_modules', 'wct-browser-legacy');
-        const version =
-            require(path.join(wctBrowserLegacyPath, 'package.json')).version;
-        if (version) {
-          browserScript = 'wct-browser-legacy/browser.js';
-        }
-      } catch (e) {
-        // Safely ignore.
-      }
-      const packageName = getPackageName(options);
-      const isPackageScoped = packageName && packageName[0] === '@';
 
+    const scripts: string[] = [], extraScripts: string[] = [];
+    const modules: string[] = [], extraModules: string[] = [];
+
+    if (options.npm) {
       // concat options.clientOptions.environmentScripts with resolved
       // ENVIRONMENT_SCRIPTS
       options.clientOptions = options.clientOptions || {};
       options.clientOptions.environmentScripts =
           options.clientOptions.environmentScripts || [];
-      options.clientOptions.environmentScripts =
-          options.clientOptions.environmentScripts.concat(
-              resolveWctNpmEntrypointNames(options, ENVIRONMENT_SCRIPTS));
 
-      if (isPackageScoped) {
+      browserScript = '';
+
+      const fromPath = path.resolve(options.root || process.cwd());
+      const npmPackageName = options.wctPackageName ||
+          ['wct-mocha', 'wct-browser-legacy', 'web-component-tester'].find(
+              (p) => !!resolveFrom.silent(fromPath, p));
```

Comment:
We need to document this lookup behavior/order

Reply:
README updated.